example_configs/onedev.md

```diff
@@ -1,8 +1,8 @@
-# Configuration for Onedev
+# Configuration for OneDev
 In Onedev, go to `Administration > Authentication Sources` and click `External Authentication`
 Select `Generic LDAP`
 
-* LDAP URL: Your lldap server's ip/hostname
+* LDAP URL: ldap:// or ldaps:// (Your lldap server's ip/hostname) :3890 or :6360
```

Comment:
It might be clearer to write it as "`ldap://lldap_ip:3890` or `ldaps://lldap_ip:6360`"

Reply:
that works; done